content/projects/infrastructure/ircbot.adoc

```diff
@@ -216,7 +216,22 @@ jenkins-admin: Delete component COMPONENT1 and move its issues to COMPONENT2
 
 === Hosting Requests
 
-*Approve and initiate a plugin/library request from JIRA*
+The IRC Bot provides hosting management commands which are used by the link:/project/teams/hosting/[Jenkins Hosting Team] to process the 
+hosting requests for new Jenkins plugins, libraries and other components.
+
+*Verify the plugin/library hosting request from Jira*
+
+This command will launch the automatic hosting request checker.
+It will verify the hosting ticket in Jira and check the links supplied in the `HOSTING` tickets.
+Then it will also verify the Maven pom.xml settings and existence of the expected files in the repository.
```

Comment:
it also does gradle as well, not sure if this should be too specific though... would prefer more generic, build system setup, naming conventions etc

Reply:
```suggestion
Then it will also verify the Maven and Gradle settings and existence of the expected files in the repository.
```

Is it better @timja ?

Reply:
sure